test/Microsoft.AspNet.FileProviders.Combined.Tests/TestUtility/MockChangeToken.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.Extensions.Primitives;
```

Comment:
Could you add the license header to the file. Also sort usings so that `System` namespaces appear first.